app/controllers/users_controller.rb

```diff
@@ -36,19 +36,12 @@ def edit
   # POST /users
   def create
     @user = User.new(user_params)
-    # Set default status and privilege
-    @user.status = "waiting"
-    @user.privilege = "student"
+    # TODO: Status is hardcoded for now until we decide what to do with it
+    @user.status = 1
 
     if @user.save
-      # Tell the UserMailer to send a welcome email after save
-      UserMailer.welcome_email(@user).deliver
-
-      # Toggle to log the user in upon sign up
-      # log_in @user
-      flash[:success] = "Please confirm email"
-
-      redirect_to(root_path)
+      flash.now[:success] = "#{@user.username} created"
```

Comment:
I think it should be just: `flash[:success]` because `flash.now[...]` is normally done when you don't redirect the user